README.md

```diff
@@ -2548,6 +2549,24 @@ You can [set up reCAPTCHA verification for iOS](https://firebase.google.com/docs
 
 This adds the `REVERSED_CLIENT_ID` from the `GoogleService-Info.plist` to the list of custom URL schemes in your Xcode project, so you don't need to do this manually.
 
+### setLanguageCode
+Sets the user-facing language code for auth operations that can be internationalized, such as sendEmailVerification() or verifyPhoneNumber(). This language code should follow the conventions defined by the IETF in BCP47.
+
+**Parameters**:
+- {string} lang - language to change, ex: 'fr' for french
+
+Example usage:
+
+```javascript
+import { FirebaseX } from '@ionic-native/firebase-x/ngx';
```

Comment:
Please can you update the example to use the plugin JS API (as the other examples do) and not the Ionic Typescript wrapper which is a separate package.

Reply:
Done :)

Reply:
Great, thanks